griptape/drivers/memory/conversation/base_conversation_memory_driver.py

```diff
@@ -6,12 +6,18 @@
 from griptape.mixins import SerializableMixin
 
 if TYPE_CHECKING:
-    from griptape.memory.structure import BaseConversationMemory
+    from griptape.memory.structure import Run
 
 
 class BaseConversationMemoryDriver(SerializableMixin, ABC):
     @abstractmethod
-    def store(self, memory: BaseConversationMemory) -> None: ...
+    def store(self, runs: list[Run], metadata: Optional[dict] = None) -> None: ...
 
     @abstractmethod
-    def load(self) -> Optional[BaseConversationMemory]: ...
+    def load(self) -> tuple[list[Run], Optional[dict]]: ...
+
+    def _get_dict(self, runs: list[Run], metadata: Optional[dict] = None) -> dict:
+        data: dict = {"runs": [run.to_dict() for run in runs]}
+        if metadata is not None:
+            data["metadata"] = metadata
+        return data
```

Comment:
`_get_dict` doesn't tell me much about why this method exists. Maybe rename to something like `_to_params`?